.gitignore

```diff
@@ -5,12 +5,13 @@ dist
 # local env files
 .env.local
 .env.*.local
-
+.env
 # Log files
 npm-debug.log*
 yarn-debug.log*
 yarn-error.log*
 pnpm-debug.log*
+yarn.lock
```

Comment:
Same here, shoulnd't be in .gitignore

Reply:
I agree with Alessandro here, these files shouldn't be ignored.